simpleLooping/test_removeSpaceAtBeginning.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from removeSpaceAtBeginning import removeSpaceAtBeginning
+
+class TestRemoveSpaceAtBeginning(unittest.TestCase):
+    def test_emptyList(self):
+        self.assertEqual(removeSpaceAtBeginning(""), "")
+
+    def test_spaceAtbeginingInput(self):
```

Comment:
What is “spaceAtBegining” mean? The other problem you make it so nice with “hasSpaceAtBegining”, so why the different?

Reply:
ok.

Reply:
Sorry, I don’t understand what you mean by “ok”? Do you agree, not agree, or planning to fix?

Reply:
Hello? Did you have a look on this one?

Reply:
Please review this.